src/Features/Core/Portable/Completion/CompletionRules.cs

```diff
@@ -20,20 +21,22 @@ internal class CompletionRules
         private readonly object _gate = new object();
         private readonly AbstractCompletionService _completionService;
         private readonly Dictionary<string, PatternMatcher> _patternMatcherMap = new Dictionary<string, PatternMatcher>();
+        private readonly Dictionary<string, PatternMatcher> _enUSCulturePatternMatcherMap = new Dictionary<string, PatternMatcher>();
```

Comment:
@davkean Does doing ordinal lookups here mean we could have a bug where someone changes their culture and we start returning cached matches against their previous culture?